ossm-release-notes/ossm-release-notes.adoc

```diff
@@ -26,7 +29,8 @@ include::modules/ossm-release-notes-3-0-deprecated-removed-features.adoc[levelof
 [id="additional-resources-release-notes_{context}"]
 == Additional resources
 
-* xref:../ossm-release-notes/ossm-release-notes-support-tables.adoc#ossm-release-notes-support-tables[Service Mesh 3.0 feature support tables]
+* xref:../ossm-release-notes/ossm-release-notes-version-support-tables.adoc#ossm-release-notes-version-support-tables[Service Mesh 3.x version support]
```

Comment:
This xref opens a page with the title, "Service Mesh version support tables". You might want to update the link text here to match that.